packages/core/src/alert/alert-group.element.ts

```diff
@@ -111,6 +105,11 @@ export class CdsAlertGroup extends LitElement {
     `;
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+    this.role = 'region';
```

Comment:
Other files we are just setting `role = 'region'` as a property on the class. Why are we doing it in `connectedCallback` here?

Reply:
I had it as a property as you describe but then hit issues with Jest, I can double check to see if the isNode check prevents the issue here